escrow/etcd_storage_test.go

```diff
@@ -165,31 +154,6 @@ func getFreePort() (port int, err error) {
 	return
 }
 
-func getTestEtcdContext(data *testPaymentData) *handler.GrpcStreamContext {
-	cleanableEtcdStorage.Put(
-		newPaymentChannelKey(data.ChannelID, data.ChannelNonce),
-		&PaymentChannelData{
-			Nonce:            big.NewInt(data.ChannelNonce),
-			State:            data.State,
-			Sender:           testPublicKey,
-			Recipient:        recipientPublicKey,
-			FullAmount:       big.NewInt(data.FullAmount),
-			Expiration:       data.Expiration,
-			AuthorizedAmount: big.NewInt(data.PrevAmount),
-			Signature:        nil,
-			GroupId:          big.NewInt(data.GroupId),
-		},
-	)
-	md := getEscrowMetadata(data.ChannelID, data.ChannelNonce, data.NewAmount)
-	return &handler.GrpcStreamContext{
-		MD: md,
-	}
-}
-
-func clearTestEtcdContext() {
-	cleanableEtcdStorage.Clear()
-}
-
 func TestEtcdGetPayment(t *testing.T) {
```

Comment:
TestEtcdGetPayment -> TestEtcdGet?

Reply:
I deleted the etcd_storage_test.go test.